docs/src/App.js

```diff
@@ -75,7 +75,7 @@ class App extends React.Component {
           <link
             key="webfont"
             rel="stylesheet"
-            href="http://fonts.googleapis.com/css?family=Montserrat:400,600|Roboto+Mono"
+            href="//fonts.googleapis.com/css?family=Montserrat:400,600|Roboto+Mono"
```

Comment:
@jxnblk Do we want to change the font weights to 500 & 700 while we're at it?

Reply:
I think we'll want to wait until v2 is ready since the font weights in the theme are still 400 and 600, right?

Reply:
True, good call. We can hold off.